packages/ui/src/components/CtlBar/index.tsx

```diff
@@ -35,10 +35,7 @@ const CtlBar = (props: Props) => {
         }}
       >
         {pageNum > 1 && (
-          <>
-            <Pager pageCursor={pageCursor} pageNum={pageNum} setPageCursor={setPageCursor} />
-            <strong style={{ color: 'white', fontSize: '0.9rem', padding: 0 }}>|</strong>
```

Comment:
I removed the ` | ` separator as I thought it looked better with just space but I can re-add